plugins/base/src/main/kotlin/transformers/documentables/ModuleAndPackageDocumentationTransformer.kt

```diff
@@ -0,0 +1,81 @@
+package org.jetbrains.dokka.base.transformers.documentables
+
+import org.jetbrains.dokka.model.DModule
+import org.jetbrains.dokka.model.PlatformDependent
+import org.jetbrains.dokka.parsers.MarkdownParser
+import org.jetbrains.dokka.plugability.DokkaContext
+import org.jetbrains.dokka.transformers.documentation.PreMergeDocumentableTransformer
+import org.jetbrains.kotlin.name.FqName
+import java.nio.file.Files
+import java.nio.file.Paths
+
+
+internal object ModuleAndPackageDocumentationTransformer : PreMergeDocumentableTransformer {
+
+    override fun invoke(original: List<DModule>, context: DokkaContext): List<DModule> {
+
+        val modulesAndPackagesDocumentation =
+            context.configuration.passesConfigurations
+                .map {
+                    Pair(it.moduleName, it.platformData) to
+                        it.includes.map { Paths.get(it) }
+                            .filter { Files.exists(it) }
```

Comment:
I'd log warning about nonexistent file here. Otherwise it'll be hard to debug

Reply:
Ok